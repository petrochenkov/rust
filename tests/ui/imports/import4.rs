mod a { pub use crate::b::foo; } //~ ERROR unresolved import `crate::b::foo`
                                 //~| ERROR cannot determine resolution for the import
mod b { pub use crate::a::foo; } //~ ERROR cannot determine resolution for the import

fn main() { println!("loop"); }
