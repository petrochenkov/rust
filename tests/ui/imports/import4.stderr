error: cannot determine resolution for the import
  --> $DIR/import4.rs:1:17
   |
LL | mod a { pub use crate::b::foo; }
   |                 ^^^^^^^^^^^^^

error: cannot determine resolution for the import
  --> $DIR/import4.rs:3:17
   |
LL | mod b { pub use crate::a::foo; }
   |                 ^^^^^^^^^^^^^

error[E0432]: unresolved import `crate::b::foo`
  --> $DIR/import4.rs:1:17
   |
LL | mod a { pub use crate::b::foo; }
   |                 ^^^^^^^^^^^^^ no `foo` in `b`

error: aborting due to 3 previous errors

For more information about this error, try `rustc --explain E0432`.
